packages/ui-default/backendlib/template.ts

```
@@ -117,94 +116,164 @@ class Nunjucks extends nunjucks.Environment {
       console.log(self);
       return self;
     });
+
+    // eslint-disable-next-line no-eval
+    this.addGlobal('eval', eval);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)